core/chain_makers.go

```diff
@@ -414,8 +415,7 @@ func (cm *chainMaker) makeHeader(parent *types.Block, gap uint64, state *state.S
 
 	if config.IsGranite(header.Time) {
 		headerExtra := customtypes.GetHeaderExtra(header)
-		timeMilliseconds := header.Time * 1000 // convert to milliseconds
-		headerExtra.TimeMilliseconds = &timeMilliseconds
+		headerExtra.TimeMilliseconds = &timeMS
```

Comment:
Does this mean that even post-granite activation, the `TimeMilliseconds` values for blocks will be round seconds?